go-controller/pkg/ovn/common.go

```diff
@@ -25,6 +25,30 @@ func hashedPortGroup(s string) string {
 	return hashForOVN(s)
 }
 
+func findAddressSet(name string, hashName string) (string, error) {
+	uuid, stderr, err := util.RunOVNNbctl("--data=bare",
```

Comment:
as we discussed moving away from using OVN  commands to find the address set and generating the name on the fly will be a better way to do get the address set name